airflow/contrib/hooks/snowflake_hook.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import snowflake.connector
+
+from airflow.hooks.dbapi_hook import DbApiHook
+
+
+class SnowflakeHook(DbApiHook):
+    """
+    Interact with Snowflake.
+
+    get_sqlalchemy_engine() depends on snowflake-sqlalchemy
+
+    """
+
+    conn_name_attr = 'snowflake_conn_id'
+    default_conn_name = 'snowflake_default'
+    support_auto_commit = True
```

Comment:
Think this is a typo, should be `supports_autocommit` to override

Reply:
Good catch, I pushed a commit that fixes this.